packages/_flutterfire_internals/lib/src/exception.dart

```diff
@@ -47,7 +47,9 @@ FirebaseException platformExceptionToFirebaseException(
 
   if (details != null) {
     code = (details['code'] as String?) ?? code;
-    message = (details['message'] as String?) ?? message;
+    message = (details['nativeErrorMessage'] as String?) ??
```

Comment:
Is it going to change a lot of error messages? Should another field like "nativeErrorMessage" in the FirebaseExeption?